docs/content/how-to/openstack/create-openstack-cluster.md

```diff
@@ -208,23 +219,45 @@ version   4.17.0        True        False         5m39s   Cluster version is 4.1
 
 ## Ingress and DNS
 
+As mentioned in the Limitations section, the Ingress and DNS configuration is a manual step that needs to be done after the cluster is created.
+This will be automated in the future.
+
 Once the workload cluster is deploying, the Ingress controller will be installed
 and a router named `router-default` will be created in the `openshift-ingress` namespace.
 
 You'll need to update your DNS with the external IP of that router so Ingress (and dependent operators like console) can work.
-You can run this command to get the external IP:
+
+Once the HostedCluster is created, you need to wait for the `router-default` service to get an external IP.
+You can check the status of the `router-default` service by running the following command:
 
 ```shell
-oc --kubeconfig $CLUSTER_NAME-kubeconfig -n openshift-ingress get service/router-default -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
+watch oc --kubeconfig $CLUSTER_NAME-kubeconfig -n openshift-ingress get service/router-default
```

Comment:
Rather than using the external command `watch`, I suggest we use the `-w` flag to watch for changes.